feature-libs/organization/order-approval/components/details/order-approval-details.module.ts

```diff
@@ -18,6 +18,7 @@ import {
   OrderDetailTotalsComponent,
   SpinnerModule,
 } from '@spartacus/storefront';
+import { ApproverGuard } from '../../core/public_api';
```

Comment:
instead of going to the public_api through relative path to access Approver guard, I believe there's a pattern set for these `feature-libs`. 

But it would be to update the tsconfig*.json to be able to use the suggested fix. Moreover, the current implementation has import errors that is causing the build to fail and I wasn't able to run the project manually. 

I believe it would be best to follow how to import is being used like how `AdminGuard` does it when they import it to certain configs for usage
```suggestion
import { ApproverGuard } from '@spartacus/organization/order-approval/core';
```

Reply:
Good catch!

That import was wrong.  However we import the guard using a relative path because the approvals feature does not have a standalone "core" entry point.  Just root, asstes, and another one for the rest.